components/organisms/SearchDialog/search-dialog.tsx

```diff
@@ -136,50 +138,68 @@ const SearchDialog = () => {
   };
 
   return (
-    <div className="fixed left-0 top-0 z-auto p-5 w-full h-full flex justify-center bg-white/30">
-      <div className="absolute w-full h-full left-0 top-0 z-50 backdrop-blur-sm" onClick={() => setOpenSearch(false)} />
-      <div
-        className="flex flex-col w-full max-w-2xl h-fit max-h-full bg-white shadow-xl border transition rounded-lg ring-light-slate-6 relative z-50 overflow-hidden"
-        onMouseMove={() => cursor !== -1 && setCursor(-1)}
-      >
-        <div className="flex w-full h-full items-center border-b p-2 pl-3">
-          {isSearching ? (
-            <div className="flex-none w-4 h-4 rounded-full border-2 border-light-slate-9 border-b-light-slate-5 border-r-light-slate-5 animate-spin" />
-          ) : (
+    <Dialog open={openSearch} onOpenChange={setOpenSearch}>
```

Comment:
It looks like more changes than it is, but TLDR; the content is wrapped in a ` <Dialog />` component now.